hles/extraction/src/main/scala/org/broadinstitute/monster/dap/RedCapClient.scala

```diff
@@ -46,6 +47,11 @@ object RedCapClient {
     val client = new OkHttpClient.Builder()
       .connectTimeout(timeout)
       .readTimeout(timeout)
+      .addInterceptor(new HttpLoggingInterceptor(new Logger() {
+        override def log(s: String): Unit = {
```

Comment:
scala syntactic sugar question - would it be possible to just directly overwrite this function, e.g. something like `override def log = logger.info`, since they have the same signature and (presumably) return type?

Reply:
🤔 That _seems_ like something scala might support, but I'm unaware of any syntactic sugar to that effect.